.gitignore

```diff
@@ -13,3 +13,5 @@
 /docs/build/
 result.tif
 result.tif.aux.xml
+pystac/*
+test.py
```

Comment:
The .gitignore looks a bit messy. Also, why having a pystac directory and not pushing it? What's the purpose of that file? What about the test.py? If this file is necessary, it should be located in the test directory.